### PR TITLE
App Hosting: Reorder cli prompt to match console experience

### PR DESCRIPTION
**Pull request by sjjj986**
Update the apphosting backends:create prompt order to match console experience

### CODE REVIEW

File: src/apphosting/githubConnections.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export async function linkGitHubRepository(
   projectId: string,
   location: string,
 ): Promise<devConnect.GitRepositoryLink> {
-  utils.logBullet(clc.bold(`${clc.yellow("===")} Set up a GitHub connection`));
+  utils.logBullet(clc.bold(`${clc.yellow("==")} Import a GitHub repository`));
```
Comment (mathu97) [inline src/apphosting/githubConnections.ts:98]:
```suggestion
  utils.logBullet(clc.bold(`${clc.yellow("===")} Import a GitHub repository`));
```

File: src/apphosting/index.ts [hunk 2]
```diff
@@ -96,13 +96,36 @@ export async function doSetup(
...
+  // enter tag:<pattern>?
+  const branch = await promptOnce({
+    name: "branch",
+    type: "input",
+    default: "main",
+    message: "Pick a branch for continuous deployment",
+  });
+  logSuccess(`Repo linked successfully!\n`);
+
+  logBullet(`${clc.yellow("==")} Setup your beckend`);
```
Comment (mathu97) [inline src/apphosting/index.ts:123]:
```suggestion
  logBullet(`${clc.yellow("===")} Setup your beckend`);
```

File: src/apphosting/index.ts [hunk 3]
```diff
@@ -236,6 +237,7 @@ async function promptNewBackendId(
       await apphosting.getBackend(projectId, location, backendId);
     } catch (err: any) {
       if (err.status === 404) {
+        logSuccess(`Name set to ${backendId}\n`);
```
Comment (mathu97) [inline src/apphosting/index.ts:240]: I think this should be logged after wherever `promptNewBackendId` is called.

**Review by mathu97 — COMMENTED** [review]
I think these logs should be removed as they seem repetitive with the new logs added:
Screenshot 2024-05-08 at 4 28 59 PM

**Review by tonyjhuang — APPROVED** [review]
🔥🔥

**Review by mathu97 — APPROVED** [review]

**Pull request merged**